app/controllers/feedbacks_controller.rb

```diff
@@ -177,4 +159,18 @@ def find_feedback_for(user)
       end
     end
   end
+
+  def execute_button_action(format)
```

Comment:
Unused method argument - `format`. If it's necessary, use `_` or `_format` as an argument name to indicate that it won't be used. You can also write as `execute_button_action(*)` if you want the method to accept any arguments but don't care about them.